common-docs/teachertool/test/catalog-shared.json

```diff
@@ -11,30 +11,74 @@
                     "name": "count",
                     "type": "number",
                     "default": 1,
-                    "path": "checks[0].count"
+                    "paths": ["checks[0].count"]
```

Comment:
Another scenario might be "A variable named ${name} is set and used". We need one check to validate the "Set" exists and one check to validate the "used" exists, but ${name} has to be inserted into both checks, so that's one parameter being put into multiple paths.

There could probably be work-arounds for this by defining more validator plans and adding the complexity there instead, but I think it's cleaner and much easier to use if we just allow the parameter to be inserted into multiple places in the json.

Reply:
Yeah, makes sense. Thanks for the example!